main.js

```diff
@@ -1,42 +1,59 @@
 const collection = document.querySelector('.collection');
-const titleInput = document.querySelector('.title-input');
-const authorInput = document.querySelector('.author-input');
+const title = document.querySelector('.title-input');
+const author = document.querySelector('.author-input');
 const addBtn = document.querySelector('.add-btn');
-let books = [];
+let books = JSON.parse(localStorage.getItem('books')) || [];
```

Comment:
- [ ] Following the project requirements, you should use JavaScript `classes` instead of `objects` and `arrays`, please note that you can use objects and arrays but it has to be **inside** the classes. For example, the variable `books` should be defined inside a class.
```javascript
class Books {
  constructor() {
    this.books = [];
  }
}
```